<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MD5_Decrypt</class>
 <widget class="QWidget" name="MD5_Decrypt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MD5解密</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/23584/Pictures/lovewallpaper/PK2.png</normaloff>C:/Users/23584/Pictures/lovewallpaper/PK2.png</iconset>
  </property>
  <widget class="QLineEdit" name="Before_Decrypt">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>解密后的值是：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始解密</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>MD5值是：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
